src/main/java/frc/robot/subsystems/AssistedBalance.java

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) FIRST and other WPILib contributors// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+ 
+package frc.robot.subsystems;
+ 
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import edu.wpi.first.wpilibj.Joystick;
+
+//import com.kauailabs.navx.frc.AHRS;
+//import edu.wpi.first.wpilibj.SPI;
+ 
+public class AssistedBalance extends SubsystemBase {
```

Comment:
Since no new hardware is introduced, don't make a new subsystem. Instead make a Command that uses the SwerveSubsystem.

Reply:
Done and everything seems to work